metaflow/runner/subprocess_manager.py

```diff
@@ -42,6 +74,18 @@ class SubprocessManager(object):
     def __init__(self):
         self.commands: Dict[int, CommandManager] = {}
 
+        try:
+
+            async def handle_sigint():
+                await self._async_handle_sigint()
+
+            asyncio.get_running_loop().add_signal_handler(
+                signal.SIGINT, lambda: asyncio.create_task(handle_sigint())
+            )
```

Comment:
Closure required in order to capture `self`, also asyncio signal handlers can't be async.